core/src/main/java/org/apache/accumulo/core/client/admin/FindMax.java

```diff
@@ -135,15 +138,17 @@ private static Text _findMax(Scanner scanner, Text start, boolean inclStart, Tex
         return next.getRow();
       }
 
-      Text ret = _findMax(scanner, next.followingKey(PartialKey.ROW).getRow(), true, end, inclEnd);
+      Text ret = _findMax(scanner,
+          RowRange.range(next.followingKey(PartialKey.ROW).getRow(), true, end, inclEnd));
       if (ret == null) {
         return next.getRow();
       } else {
         return ret;
       }
     } else {
 
-      return _findMax(scanner, start, inclStart, mid, mid.equals(start) ? inclStart : false);
+      return _findMax(scanner,
+          RowRange.range(start, inclStart, mid, mid.equals(start) && inclStart));
```

Comment:
This is fine, but I think the ternary expression is a little more readable than the boolean conjunction.